pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -2188,18 +2189,19 @@ func TestComputePodActionsForPodResize(t *testing.T) {
 	memPolicyRestartRequired := v1.ContainerResizePolicy{ResourceName: v1.ResourceMemory, RestartPolicy: v1.RestartContainer}
 
 	for desc, test := range map[string]struct {
-		podResizePolicyFn       func(*v1.Pod)
-		mutatePodFn             func(*v1.Pod)
+		setupFn                 func(*v1.Pod, *kubecontainer.PodStatus)
 		getExpectedPodActionsFn func(*v1.Pod, *kubecontainer.PodStatus) *podActions
 	}{
 		"Update container CPU and memory resources": {
-			mutatePodFn: func(pod *v1.Pod) {
-				pod.Spec.Containers[1].Resources = v1.ResourceRequirements{
+			setupFn: func(pod *v1.Pod, status *kubecontainer.PodStatus) {
```

Comment:
not exactly. since we're no longer fetching the actual resources from `pod.Status`, we need to configure the `kubecontainer.PodStatus` here, so I renamed the function accordingly.

Collapsing `podResizePolicyFn` into `setupFn` is just cleanup. I don't know why that had its own function. This test was a bit of a mess, TBH.